packages/runtime/container-runtime/src/dataStore.ts

```diff
@@ -12,6 +12,13 @@ import { TelemetryDataTag } from "@fluidframework/telemetry-utils";
 import { ContainerRuntime } from "./containerRuntime";
 import { DataStores } from "./dataStores";
 
+/**
+ * @deprecated - This will be removed once https://github.com/microsoft/FluidFramework/issues/9127 is fixed.
+ */
+export interface IDataStoreWithBindToContext_Deprecated extends IDataStore {
```

Comment:
should this be in definitions?

Reply:
Hmmm, this won't be exported from here. I can export this explicitly.
I didn't add it to definitions because its temporary and removing it would take longer from definitions because of the compat validations.

Reply:
i'd rather they not need a reference on this package. the consumer here right now probably only depends on definitions